health-integrations/README.md

```diff
@@ -14,3 +15,27 @@ InjectorBuilder
     )
     .createInjector();
 ```
+
+### Archaius2-backed IndicatorFilter
+Filtering of which HealthIndicators will be used in communicating the health status of the application is provided via an [Archaius2](https://github.com/Netflix/archaius/tree/2.x) implementation of [IndicatorFilter](https://github.com/Netflix/runtime-health/blob/addIndicatorFilterSupport/health-api/src/main/java/com/netflix/runtime/health/api/IndicatorFilter.java).
+
+```
+InjectorBuilder
+    .fromModules(
+        // ... other modules ...
+        new HealthModule(),
+        new EurekaHealthStatusBridgeModule(),
+        new ArchaiusHealthStatusFilterModule(),
+        // ... other modules ...
+    )
+    .createInjector();
+```
+
+Indicators can now be included/excluded by setting the following properties in Archaius2.
+```
+health.status.indicators.include=com.myproject.MyFilter
+health.status.indicators.exclude=com.myproject.DontIncludeThisFilter
```

Comment:
Excludes do take priority if you've included a large set and only want to exclude one for some reason. But in general I expect people to only want to use one. Up to you to decide which way you prefer. Including only your own would give you the behavior of not getting any other indicators if that is what you want.